### DISCUSSION TITLE
There is no permission named `viewAnyRole` for guard `web`.

### DISCUSSION THREAD

**Discussion by lekesiz** [spatie/laravel-permission#2392] [Q&A]
Spatie \ Permission \ Exceptions \ PermissionDoesNotExist 
PHP 8.2.4
10.6.2
There is no permission named `viewAnyRole` for guard `web`.

**Comment by drbyte** [comment]
Hi,
If you are looking for help with debugging your application, please share your application code. The best way is to create a simple new application which exposes this specific problem, and push that application to a github repo.
Or at least show what commands and/or application code are triggering the error.
